src/cpx_model.jl

```diff
@@ -240,3 +240,55 @@ function set_terminate(model::Model)
 end
 
 terminate(model::Model) = (model.terminator[1] = 1)
+
+mutable struct ConflictRefinerData
+    stat::Int
+    nrows::Int # number of rows participating in the conflict
+    rowind::Vector{Cint} # index of the rows that participate
+    rowstat::Vector{Cint} # state of the rows that participate
+    ncols::Int # number of columns participating in the conflict
+    colind::Vector{Cint} # index of the columns that participate
+    colstat::Vector{Cint} # state of the columns that participates
+end
+
+function refineconflict(model::Model)
+    # First, compute the conflict. 
+    confnumrows_p = Vector{Cint}(undef, 1)
+    confnumcols_p = Vector{Cint}(undef, 1)
```

Comment:
To avoid creating a one-element vector, you can use 
```julia
confnumcols_p = Ref{Cint}()
```
Then you can access the value by `confnumcols_p[]`